### PR TITLE
Announce 0.9.901

### PR DESCRIPTION
**Pull request by tbkka**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,9 +4,13 @@
 
 ---
 
-> :warning: **WARNING** :warning: This project is in a _prerelease_ state. There
```
Comment (thomasvl) [inline README.md]: I think this block is in a bunch of the Documentation files, update those also?

File: README.md [hunk 2]
```diff
@@ -134,7 +138,7 @@ If you're using CocoaPods, add this to your `Podfile` but adjust the `:tag` to
 match the `[tag_name]` you used to build the plugin above:
 
 ```ruby
-pod 'SwiftProtobuf', git: 'https://github.com/apple/swift-protobuf.git', :tag => '0.9.24'
+pod 'SwiftProtobuf', git: 'https://github.com/apple/swift-protobuf.git', :tag => '0.9.901'
```
Comment (thomasvl) [inline README.md]: `DevTools/LibraryVersions.py 0.9.901` to update the version numbers?

File: README.md (hunk 1 above)
Comment (tbkka) [inline README.md]: Done.

File: README.md (hunk 2 above)
Comment (tbkka) [inline README.md]: Done.

**Comment by tbkka** [comment]
I just went through and edited a bunch of the method and property names mentioned in the documentation -- I've gotten a few queries from folks who were confused by the mismatch.

File: Documentation/API.md [hunk 3]
```diff
@@ -230,22 +197,17 @@ Notes: For performance, the field may be initialized lazily, but this is
 invisible to the user.  The property will be serialized if it has a non-default
 value.
 
-**Proto2 `optional` fields without default values** generate properties whose
-type is `Optional<T>` for the corresponding T as listed above.  The initial
-value is nil.  You can set the value to nil to erase it.
+**Proto2 `optional` fields** generate properties of the corresponding
+type above. TODO  You can set the value to nil to erase it.
```
Comment (thomasvl) [inline Documentation/API.md]: This is wrong, we have clear* methods.

Comment (tbkka) [inline Documentation/API.md]: Changed.

**Pull request merged**